unicode.go

```diff
@@ -14,23 +14,15 @@ usage: unicode [-c] [-d] [-n] [-t]
 	-U: output full Unicode description
 
 Default behavior sniffs the arguments to select -c vs. -n.
-
-For some options you will need UnicodeData.txt installed.
-Use curl or wget or your favorite webirific tool to copy
-	ftp://ftp.unicode.org/Public/UNIDATA/UnicodeData.txt
-to
-	$GOPATH/src/robpike.io/cmd/unicode
 */
 package main // import "robpike.io/cmd/unicode"
 
 import (
-	"bufio"
 	"bytes"
+	_ "embed"
```

Comment:
this being a different class of thing, i'd expect it to be its own paragraph inside the import list, like an external module. but gofmt doesn't do that. humph.